index.html

```diff
@@ -4,12 +4,14 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="style.css">
```

Comment:
Could you leave the css into a folder called `css` or `styles` as you please, think that in the future you will add more css files or assets and it'll be good to have them there